webpack.build.config.js

```diff
@@ -12,9 +12,22 @@ if (!isProd) {
   /*
   Till we do not import css from react components, we need to have
   this workaround for Hot reload to work with global css.
+  We are not using "glob" to get all files in "css" folder as order of files matters.
   */
-  const globalCssPathnames = glob.sync('./app/public/css/**/*.css') || []
-  entry = entry.concat(globalCssPathnames)
+  const cssPathnames = [
+    './app/public/css/styles.css',
+    './app/public/css/0-767px.css',
+    './app/public/css/0-1366px.css',
+    './app/public/css/767-1366px.css',
+    './app/public/css/768-1024px.css',
+    './app/public/css/1024-1112px.css',
+  ]
+  // Check if "cssPathnames" are up-to-date with css files stored in "app/public/css" folder
+  if (JSON.stringify([...cssPathnames].sort()) !== JSON.stringify(glob.sync('./app/public/css/**/*.css').sort())) {
```

Comment:
@refi93 I did not find it necessary to write some array equals function for this. Do you think it should be there?

Reply:
not really, that was just my first thought about approaching it, this is does the job as well and is simpler, so looks good to me